src/UNIV2LPOracle.t.sol

```diff
@@ -331,7 +331,9 @@ contract UNIV2LPOracleTest is DSTest {
 
     // Passed 10 rounds of fuzzing with 10,000 test cases
     function test_compare_sqrt(uint256 exp) public {
-            // Convert to WAD since that is what we operate on
+        if (exp == 0) return;
```

Comment:
Good addition here heh

Reply:
This edge case was actually making the tests flaky...so I snuck it into this PR ;).